### PR TITLE
Revert #20528 (dependency verification key de-duplication)

### PR DESCRIPTION
**Pull request by hegyibalint**

### CODE REVIEW

**Comment by hegyibalint** [comment]
@bot-gradle test RFM please

**Comment by hegyibalint** [comment]
@ljacomet I'm unsure: this should point to `master` or `release`?

**Comment by ljacomet** [comment]
This needs to be done against `release`

**Review by ljacomet — CHANGES_REQUESTED** [review]
This needs to target `release` and should also come with a complete re-generation of the keys.

**Comment by hegyibalint** [comment]
@bot-gradle test RFM please

File: subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc [hunk 1]
```diff
@@ -33,134 +33,6 @@ Some plugins will break with this new version of Gradle, for example because the
...
-[[changes_7.6]]
-== Upgrading from 7.5 and earlier
-
-[[exported_key_deduplication]]
-=== Exported keys are now deduplicated
-
-link:dependency_verification.html#sec:local-keyring[Dependency verification's] way of key serialization was changed in 7.6: keys now are sorted and deduplicated.
-Forward compatibility was retained, but the following exports might cause a large VCS diff between the old, and the new keyfile.
-
-=== Updates to Attribute Disambiguation Rules related methods
```
Comment (ljacomet) [inline subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc:45]: You are reverting way too many things in there. Only the section about export needs to be removed.

**Review by ljacomet — CHANGES_REQUESTED** [review]

File: subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc (hunk 1 above)
Comment (hegyibalint) [reply subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc:45]: Correct, hang on...

Comment (hegyibalint) [reply subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc:45]: PTAL

**Review by ljacomet — APPROVED** [review]

File: gradle/verification-metadata.xml [hunk 2]
```diff
@@ -507,6 +507,11 @@
             <pgp value="47504b76cf89c15c0512d9afe16ab52d79fd224f"/>
          </artifact>
       </component>
+      <component group="com.google.code.gson" name="gson" version="2.8.6">
+         <artifact name="gson-2.8.6.jar">
+            <pgp value="afcc4c7594d09e2182c60e0f7a01b0f236e5430f"/>
```
Comment (hegyibalint) [inline gradle/verification-metadata.xml:512]: Checking this one, just to be sure.

Comment (hegyibalint) [reply gradle/verification-metadata.xml:512]: Just had a look, as the UID was a bit weird.
The packet is:
```
# off=0 ctb=c6 tag=6 hlen=3 plen=269 new-ctb
:public key packet:
	version 4, algo 1, created 1448391204, expires 0
	pkey[0]: [2048 bits]
	pkey[1]: [17 bits]
	keyid: 7A01B0F236E5430F
# off=272 ctb=cd tag=13 hlen=2 plen=43 new-ctb
:user ID packet: "Inderjeet Singh <inder@alumni.stanford.edu>"
# off=317 ctb=c2 tag=2 hlen=3 plen=312 new-ctb
:signature packet: algo 1, keyid 7A01B0F236E5430F
	version 4, created 1448391204, md5len 0, sigclass 0x13
	digest algo 2, begin of digest ea bf
	hashed subpkt 2 len 4 (sig created 2015-11-24)
	hashed subpkt 27 len 1 (key flags: 03)
	hashed subpkt 11 len 5 (pref-sym-algos: 9 8 7 3 2)
	hashed subpkt 21 len 5 (pref-hash-algos: 8 2 9 10 11)
	hashed subpkt 22 len 3 (pref-zip-algos: 2 3 1)
	hashed subpkt 30 len 1 (features: 01)
	hashed subpkt 23 len 1 (keyserver preferences: 80)
	subpkt 16 len 8 (issuer key ID 7A01B0F236E5430F)
	data: [2045 bits]
# off=632 ctb=ce tag=14 hlen=3 plen=269 new-ctb
:public sub key packet:
	version 4, algo 1, created 1448391204, expires 0
	pkey[0]: [2048 bits]
	pkey[1]: [17 bits]
	keyid: C3E640F38D845FA2
# off=904 ctb=c2 tag=2 hlen=3 plen=287 new-ctb
:signature packet: algo 1, keyid 7A01B0F236E5430F
	version 4, created 1448391204, md5len 0, sigclass 0x18
	digest algo 2, begin of digest 6c 63
	hashed subpkt 2 len 4 (sig created 2015-11-24)
	hashed subpkt 27 len 1 (key flags: 0C)
	subpkt 16 len 8 (issuer key ID 7A01B0F236E5430F)
	data: [2048 bits]
```
The ASC file, the keyid agrees, and everything checks out, so this is good to go.

**Review by ljacomet — APPROVED** [review]

**Comment by ljacomet** [comment]
@bot-gradle test and merge

**Pull request merged**